executor/replace_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package executor_test
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/util/collate"
+	"github.com/pingcap/tidb/util/testkit"
+)
+
+func (s *testSerialSuite) TestIssue20849(c *C) {
```

Comment:
Can you move this to the `write_test.go`? I saw many existing tests about `REPLACE INTO` are there.